adoc/admin-security-ugmgmt.adoc

```diff
@@ -0,0 +1,236 @@
+= Managing Users and Groups
+
+You can use standard LDAP administration tools for managing these users remotely.
```

Comment:
You can use standard LDAP administration tools for managing these **organizations, groups and users** remotely.

Reply:
Also, please change the title of PR to `User group management`